packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customReducers.esm.t

```diff
@@ -9,6 +9,7 @@ unless_exists: true
 */
 
 import { <%= h.changeCase.pascal(documentType) %><%= h.changeCase.pascal(module) %>Operations } from '../../gen/<%= module %>/operations';
+import { default as Exceptions } from '../../gen/<%= module %>/exceptions';
```

Comment:
I wouldn't import the errors here yet since we are not doing anything with them.

Right now, the Operation type has a error?: string; property, it might be helpful in the future to have a way to infer the errors from each action type. But not something for now.